<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupManager</class>
 <widget class="QWidget" name="GroupManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>群组管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>群组管理</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="createLayout">
     <item>
      <widget class="QLineEdit" name="groupNameEdit">
       <property name="placeholderText">
        <string>群组名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="groupDescEdit">
       <property name="placeholderText">
        <string>群组描述</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>创建群组</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>刷新列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="joinButton">
       <property name="text">
        <string>加入群组</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="leaveButton">
       <property name="text">
        <string>退出群组</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="groupListWidget"/>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string></string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
